src/Goerli-DssSpell.sol

```diff
@@ -31,47 +31,34 @@ contract DssSpellAction is DssAction {
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
```

Comment:
I think we can remove the `DssVestLike` interface in L22 as not used in the spell.